### PR TITLE
CRW-795 list installed RPMs so we can audit...

### PR DESCRIPTION
**Pull request by nickboldt**
CRW-795 list installed RPMs so we can audit versions more easily; also update to latest base images and fix CVE issue in systemd

Change-Id: Ia65157e5622b62acb3051f22e9dccc3e5021df49
Signed-off-by: nickboldt <nboldt@redhat.com>

### CODE REVIEW

**Comment by nickboldt** [comment]
Test died trying to
```
+ curl 'http://admin.ci.centos.org:8080/Node/fail?key=8dc1a710-6097-11e6-a671-525400ea212d&ssid=053bb5ec'
```

But the same change over here passed... can someone just approve this?

https://github.com/eclipse/che-devfile-registry/pull/227

File: build/dockerfiles/rhel.Dockerfile [hunk 1]
```diff
@@ -12,8 +12,8 @@
 
 # Builder: check meta.yamls and create index.json
 # https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8-minimal
-FROM registry.access.redhat.com/ubi8-minimal:8.1-398 as builder
-USER 0
+FROM registry.access.redhat.com/ubi8-minimal:8.1-409 as builder
+USER 0
```
Comment (benoitf) [inline build/dockerfiles/rhel.Dockerfile:16]: trailing space to fix ?

Comment (nickboldt) [reply build/dockerfiles/rhel.Dockerfile:16]: tried to retrigger the test :D

**Comment by nickboldt** [comment]
thanks!

**Pull request merged**